swift_browser_ui_frontend/src/components/FolderOptionsMenu.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <c-menu
+    simple
+    :items.prop="menuItems.map(item => ({name: $t(item.key), ...item}))"
+  >
+    <c-button
+      class="menu-trigger-button"
+      text
+      tabindex="-1"
+      :inverted="selected"
+    >
+      ... {{ $t('message.options') }}
```

Comment:
I think using an icon for dots might look better than `...` ?

Reply:
Good point! Dots are now converted to MDI icon.